app/src/lib/is-empty-or-whitespace.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Indicates whether the given string is null, undefined, empty, or consists
+ * only of white-space characters.
+ */
```

Comment:
I guess `null` and `undefined` are not options anymore, right?
```suggestion
/**
 * Indicates whether the given string is empty or consists only of white-space
 * characters.
 */
```